### PR TITLE
l10n_jp_address_layout

### PR DESCRIPTION
**Pull request by TimLai125**

### CODE REVIEW

**Comment by TimLai125** [comment]
@yostashiro Please review the PR and let me know if there is anything needed to improve.

File: l10n_jp_address_layout/models/res_country_state.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Copyright 2018 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ResCountryState(models.Model):
+    _inherit = 'res.country.state'
+
+    name = fields.Char(translate=True)
```
Comment (yostashiro) [inline l10n_jp_address_layout/models/res_country_state.py]: Let's separate the translation of the state field into a separate module `l10n_jp_country_state` which should depend on `base_country_state_translatable`.  Please check and approve the PR https://github.com/OCA/partner-contact/pull/537 by the way.

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
-# quartile-addons
+Quartile Limited Odoo-Addons
+============================
+
+Available addons
+----------------
+Addon | Version | Summary
+--- | --- | ---
+l10n_jp_address_layout | 11.0.1.0.0 | Japan Address Layout
```
Comment (yostashiro) [inline README.md]: Maybe we don't want to have this here as we will probably forget to update it.

File: l10n_jp_address_layout/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    'category': 'Extra Tools',
+    'license': "AGPL-3",
+    'description': """
+This module provides the following features:
+
+- Japan address input field layout
+- Translations of 都道府県
+    """,
+    'summary': "",
+    'depends': [
```
Comment (yostashiro) [inline l10n_jp_address_layout/__manifest__.py]: `base`, or has it been a common practice to omit it if it's `base` (in OCA)?

File: l10n_jp_address_layout/data/res_country_data.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="base.jp" model="res.country">
+        <field name="address_view_id" ref="l10n_jp_address_layout.res_partner_japan_address_form"/>
```
Comment (yostashiro) [inline l10n_jp_address_layout/data/res_country_data.xml]: Do we need `l10n_jp_address_layout.`? I wasn't sure.

**Pull request merged**